src/connectors/README.md

```diff
@@ -13,8 +13,8 @@ Connects provided component with a wrapper behind internal PWA routing actions.
 Everywhere as a React component rendered within the PWA app.
 
 #### Props provided
-- `historyPush(pathname: string)` - Opens new page in a safe way. Detects link types. Internal links (deep links) which are handled by the PWA app. External links are handled by opening the in-app-browser.
-- `historyReplace(pathname:string)` - Replaces current page with provided pathname if pathname is an internal link. If pathname is external link it works exactly as `historyPush`. If you intend to replace your current page with in-app-browser, use combination of `historyPush` and `historyPop` instead.
+- `historyPush(params: object)` - Opens new page in a safe way. Detects link types. Internal links (deep links) which are handled by the PWA app. External links are handled by opening the in-app-browser.
```

Comment:
You cannot change existing function signature like this since it would be a breaking change to all existing installation that are using this package.

Please change those by adding new attribute as we talked off line.
So for history push it would be:
`historyPush(pathname: string, params: object[optional])`

Reply:
Changed to pass pathname, state, and silent since these are the only key values the router can take in the passed object.

Reply:
Please change those by adding new attribute as we talked off line.
So for history push it would be:
historyPush(pathname: string, params: object[optional])

Inside params use `state` and `silent`. Since both are optional, it's better to not have them as arguments.

Reply:
Updated to params with a default values for state and silent.